### ISSUE TITLE
Remembering playback time doesn't work

### ISSUE THREAD

**Issue by doumeem** [iina/iina#4420]
**System and IINA version:**

- macOS  10.15.7 (Catalina)
- IINA  1.3.1

**Expected behavior:**
When opening another file or playing the next file in the playlist the app should remember the playback time for the previous file

**Actual behavior:**
After opening or playing the next file in the playlist and then going back to the previous/original file it starts from the beginning

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**
Always (?)

**Comment by low-batt** [comment]
This part of the report:
> -  [x] `MPV does not have this problem`

Is confusing as IINA's `Resume last playback position` enables mpv's [Watch Later](https://mpv.io/manual/stable/#watch-later) feature. From the mpv manual:

> Always save the current playback position on quit. When this file is played again later, the player will seek to the old playback position on start. This does not happen if playback of a file is stopped in any other way than quitting. For example, going to the next file in the playlist will not save the position, and start playback at beginning the next time the file is played.
>
> This behavior is disabled by default, but is always available when quitting the player with Shift+Q.

If you close the window or quit IINA the playback position is saved, but for some reason mpv decided that if you switch to another video in the playlist you don't want the position saved. There are a number of issues asking for changes to this feature in the [mpv](https://github.com/mpv-player/mpv/issues) GitHub project.

**Comment by doumeem** [comment]
Yeah looks like it is only playing the next file in the playlist, when quitting it's fine.
So are you guys gonna improve IINA or wait for mpv to do the change ?
(Btw i accidentally clicked the check mark)

**Comment by low-batt** [comment]
In general IINA strongly resists making changes to mpv behavior. For the case at hand, I'm thinking this is a grey area as mpv provides the command [write-watch-later-config](https://mpv.io/manual/stable/#command-interface-write-watch-later-config):
> Write the resume config file that the quit-watch-later command writes, but continue playback normally.

I would say the next step would be to review the mpv watch later issues and see what concerns the mpv team has about changing this behavior.

I'm a newbie junior developer, the senior developers will have to decide if IINA should deviate from mpv behavior in this case.
